modules/configuration/ajax/process.php

```diff
@@ -59,7 +75,45 @@
     }
 }
 
+/**
+ * Query DB for config settings that correspond to filepaths
+ *
+ * @return array IDs corresponding to path settings.
+ */
+function getPathIDs()
+{
+    $ids = array();
+    $DB  = \Database::singleton();
+    /* Query adapated from:
+     * https://github.com/aces/Loris/wiki/Project-Customization
+     */
+    $query  = "SELECT c.ID FROM Config c "
+        . "LEFT JOIN ConfigSettings cs ON (c.ConfigID = cs.ID) "
+        . "JOIN ConfigSettings csp ON (cs.Parent = csp.ID) "
+        . "WHERE csp.Name = 'paths';";
```

Comment:
I don't think just grabbing all config settings based on the name of the section is going to work. You don't know the intended use of the path (or even that it's a path and not some other type specified in the section), it could be a path for imaging scripts, there could be config options that aren't paths in the section of that name, or there could be config options that are paths which someone added to another section.

To do this right there needs to be (at least) 1 DataType added to ConfigSettings.DataType to indicate that a setting is intended to be a path, probably 2 (one for paths used be the web server one for arbitrary paths that don't get validated.)